examples/realtime_ctlog_tasker.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import sys
+import datetime
+import certstream
+import json
+import requests
+
+# This is an example of a long-running service/process which will monitor for
+# CT Logs in real-time and as soon as a certificat_update action is triggered
+# for a domain pattern we care about, we will immediately take action to task
+# port scans and observatory scans by calling our public REST API endpoints
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def print_callback(message, context):
+    logging.debug("Message -> {}".format(message))
+
+    if message['message_type'] == "certificate_update":
+        all_domains = message['data']['leaf_cert']['all_domains']
+
+        domain_patterns = [
+            ".mozilla.com",
+            ".mozilla.org",
+            ".firefox.com",
+            ".com",  # for testing purposes only, remove before setting live
+        ]
+
+        for fqdn in all_domains:
+            for domain_pattern in domain_patterns:
+                # We want all legit FDQNs, but we can't scan wild-cards
+                if fqdn.endswith(domain_pattern) and ('*' not in fqdn):
+
+                    # TODO: add relevant tasking call here to /ondemand/portscan
+                    logger.info("Sends POST to https://<YOUR-API-ENDPOINT>/dev/ondemand/portscan")
```

Comment:
I feel like the logic that will live here is not specific to CTLog tasking, but tasking in general to the REST API.  Because of this, I  think we should tackle tasking to REST API endpoints in a generic class and simply re-use it in all tasking models (ondemand, real-time, and scheduled).

Reply:
This is why I did not include it, because it seemed unnecessary to demonstrate how the CTLog watching process will be able to task a scan for a FQDN.

Reply:
Makes sense, but I may need your help implementing this.